api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
 
 # Create your models here.
+
+class Task (models.Model):
+
+	STATUS_CHOICES = (
+		('UNASSIGNED', 'Unassigned'),
+		('WORKINGON', 'Working On'),
+		('COMPLETED', 'Completed'),
+		)
+
+	title = models.CharField(max_length=255)
+	description = models.TextField(max_length=1000, blank=True)
+	status = models.CharField(max_length=15, choices=STATUS_CHOICES, default='UNASSIGNED')
+	#assigned_id = models.ForeignKey(User)
```

Comment:
This was because of the migration made locally for testing the table in the admin. I really need to uncomment now...